extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -165,6 +165,27 @@ const buildExpandedViewOverlay = element => htmlFor(element)`
       </span>
     </div>`;
 
+
+/**
+ * Updates embed's corresponding <style> element with embedData.
+ * @param {!Element} embedStyleEl
+ * @param {!Object} embedData
+ * @param {boolean} expanding
+ */
+function updateEmbedStyleEl(embedStyleEl, embedData, expanding) {
+  const embedId = embedData.id;
+
+  embedStyleEl.textContent = `[${EMBED_ID_ATTRIBUTE_NAME}="${embedId}"] {
+      width: ${px(embedData.width)} !important;
+      height: ${px(embedData.height)} !important;
+      transform: ${expanding ? 'translate3d(' + embedData.translateX + 'px, ' +
```

Comment:
This feels a bit like it's the wrong level to include data about `expanding` vs. not.  I think the caller should just take this into consideration when constructing the `embedData`.

Reply:
Done.

Reply:
What you have here LGTM, and I don't think you need to change it.

But, just to let you know, I didn't necessarily mean collapsing the transform attributes into one; the transform attributes can still exist separately, but what I meant was just that the `expanding` attribute really is a part of the expand/collapse animation specifically, whereas this function only is meant to handle turning the `embedData` JS object into a string and putting it in the `<style>` element.

It shouldn't need to know about _why_ that's happening, but just that it is.  For that reason, it seems like leaky scope to make this have to know about the `expanding` state.